lib/interactor/contracts/forwarder.rb

```diff
@@ -0,0 +1,36 @@
+module Interactor
+  module Contracts
+    # Enables a forwarding to context.params to avoid boilerplate code like `context.param`
+    module Forwarder
+      # Called when the module is extended
+      #
+      # @api private
+      # @param [Module] extended
+      # @return [void]
+      def self.extended(extended)
+        extended.extend(Forwardable) unless Forwarder.using_activesupport?
+      end
+
+      # Define delegates from all expectations to context object
+      #
+      # @api private
+      # @params [Array] all attributes keys defined in expects block
+      # @return [void]
+      def def_delegates_to_context(keys)
+        if Forwarder.using_activesupport?
```

Comment:
What is the value you find in being able to use ActiveSupport for this?

I would prefer always using `Forwardable` since it's in the standard library and works just as well. I just worry that the split in behavior based on whether you have another gem installed will be the source of some confusing bugs.